tsl/test/expected/dist_join-12.out

```diff
@@ -0,0 +1,173 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
Would be good to have a more specific name for the test, like `dist_ref_table_join`.

Reply:
The files are renamed to `dist_ref_table_join` as suggested.